Marlin/src/pins/pins_MKS_SBASE.h

```diff
@@ -104,14 +106,16 @@
 
 //
 // Connector J7
+// Note: These pins are all shared with the EXP1/EXP2 Connector.
+// Using them with an LCD connected or configured will lead to hangs & crashes.
```

Comment:
May be worth noting that although the cpu pins are shared, the J7 and EXP1 pins are not directly connected. The pins on EXP1 go via a level shifter (as do many of the pins on EXP1/EXP2).

Reply:
I think what you have is fine. Perhaps just add. "The connection to EXP1 goes via a level shifter."

I was a little confused by your original comment as I'd previously tested the J7 pins with a meter to see if they were shared with any other external connections and they did not seem to be. It was only when digging deeper that I noticed there is a level shifter on the EXP1 connection which means they do not show as shared with a simple meter check. Might save someone else the same confusion.

Reply:
added digitally, if that doesn't make enough sense I'll just copy the above.

Reply:
No that's fine!